traveloka/pod-template
# Add XCMetrics to Pod Template

## Summary

- Add XCMetrics to `Swift` and `Objc` Template
- Replace `PROJECT_DIR` with `SRCROOT` becuase `PROJECT` will be replaced with Modules name by pod template when create new module

## Test Plan
- Create new module in Traveloka/Modules with template URL git@github.com:dwirandytlvk/pod-template.git
- Run Sandbox
- See the XCMetrics generated logs in Library/Caches/XCMetrics
![Screen Shot 2022-01-26 at 14 52 26](https://user-images.githubusercontent.com/93510180/151123776-5d865995-a6c7-434d-bb25-8021f47d7a31.png)
